modular_skyrat/modules/customization/modules/surgery/organs/moth_antennae.dm

```diff
@@ -0,0 +1,3 @@
+/obj/item/organ/external/antennae
```

Comment:
Seems like I forgot to tick this one, but it was still working without it. Somehow.